src/java.base/share/classes/java/lang/reflect/Constructor.java

```diff
@@ -165,6 +165,14 @@ Constructor<T> copy() {
         return res;
     }
 
+    /** Creates a new root constructor with a custom accessor for serialization hooks. */
```

Comment:
Preserve "/**" comments for generated javadoc, a "//" comment would be more appropriate for a 1-liner.